src/Orleans/Configuration/ClientConfiguration.cs

```diff
@@ -175,7 +175,11 @@ public ClientConfiguration()
             NetInterface = null;
             Port = 0;
             DNSHostName = Dns.GetHostName();
+#if NETSTANDARD
+            DeploymentId = Environment.MachineName;
+#else
```

Comment:
I think we should set it to empty string in both cases, just like we do for `DataConnectionString`.